runtime/crab/src/pallets/polkadot_xcm.rs

```diff
@@ -121,7 +121,7 @@ impl xcm_executor::Config for XcmExecutorConfig {
 	type RuntimeCall = RuntimeCall;
 	type SubscriptionService = PolkadotXcm;
 	type Trader = UsingComponents<
-		ConstantMultiplier<Balance, ConstU128<{ BASE_WEIGHT_FEE }>>,
+		ConstantMultiplier<Balance, ConstU128<GWEI>>,
```

Comment:
Not sure this is correct replacement, BASE_WEIGHT_FEE could be changed to being not equal to 1 GWEI

Reply:
Yes, we should keep it for later adjustments. @AurevoirXavier

Reply:
IMO, `BASE_WEIGHT_FEE` is not a core primitive.

So I define it like other parameters as well.

e.g. `SessionPeriod = Constu32<{ 6 * HOURS }>`

Reply:
How about changes to `ConstU128<{1 * GWEI}>`

Reply:
https://github.com/darwinia-network/darwinia-2.0/pull/68/commits/ef0c83b8d83610156f8e749eb15cf8ea304e872a